package.json

```diff
@@ -30,7 +30,8 @@
     "lint": "tsdx lint src test --max-warnings 0",
     "prepare": "yarn build",
     "size": "size-limit",
-    "analyze": "size-limit --why"
+    "analyze": "size-limit --why",
+    "depcheck": "depcheck --ignores=\"eslint,eslint-*,babel-*,tslib,typescript,@typescript-eslint/*,@upleveled/*,@size-limit/*\""
```

Comment:
The scripts here should not be for the preflight Checks. Checks should be self-contained, with all information and resources in the JS file.

The scripts in `package.json` are for the `preflight` package - eg. testing, linting, building, etc.